### DISCUSSION TITLE
Thank you for the ONNX

### DISCUSSION THREAD

**Discussion by JosefKuchar** [RapidAI/RapidLatexOCR#3] [Ideas]
Thanks to your onnx model I was able to make full inbrowser version (https://github.com/JosefKuchar/LaTeX-OCR-Web, https://josefkuchar.com/LaTeX-OCR-Web/)
It's still WIP, but it kinda works - I need to implement model caching using CacheAPI, because loading times are kinda ridiculous (~2 minutes)

**Comment by SWHL** [comment]
Just look your project, it is a charm. 
I am looking forward to your project completion.
I'd be pleasure to help you if I could.
